201-asev2-ilb-with-web-app/azuredeploy.parameters.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "aseName": {
+      "value": "GEN-UNIQUE"
+    },
+    "vnetResourceName": {
+      "value": "GET-PREREQ-existingVnetResourceId"
+    },
+    "subnetName": {
+      "value": "GET-PREREQ-existingSubnetName"
+    },
+    "aseLocation": {
+      "value": "West Europe"
+    },
+    "internalLoadBalancingMode": {
+      "value": 3
+    },
+    "dnsSuffix": {
+      "value": "GEN-UNIQUE-10"
+    },
+    "siteName": {
+      "value": "GEN-UNIQUE"
+    },
+    "appServicePlanName": {
+      "value": "GEN-UNIQUE"
+    },
+    "workerPool": {
```

Comment:
remove this and use the defaultValue in the template

Reply:
How should this be done. In a normal situation this property isn't by the defaultvalue

Reply:
use only GEN values in the parameters file - remove those that  can use a defaultValue